sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -1,7 +1,15 @@
 # Change Log azure-ai-textanalytics
 
-## 1.0.0b5 (Unreleased)
+## 1.0.0b6 (2020-05-27)
```

Comment:
This is because I erroneously released 1.0.0b5 when trying to generate the private bits (very embarassing), so b5 is burned for python